### DISCUSSION TITLE
Details regarding the Plugin

### DISCUSSION THREAD

**Discussion by derkoe** [porscheinformatik/sonarqube-licensecheck#259] [Q&A]
@vinodkumar4b9 wrote in #258

> Regarding this Plugin Can you please provide us some details here , So Basically once you install the plugin and then based on > the project we work we need to add the licenses .
> 
> So Here Based on the licenses we add it will show you the details whether this has been allowed or not ryt ? and if you don't have any details regarding that It will show the status as Unknown . Is my understanding correct ? or Is there any way that this can say that the version we are using is not the latest or updated one some thing like that
> 
> Can you please give us a brief how this plugin exactly works so that I will be useful for me to proceed further

**Comment by derkoe** [comment]
This plugin reads all the dependencies of your project, checks all the licenses (and maps some if not found). Then it will compare this with the allowed license list (which you can maintain in the settings). Them, it generates a violation for each disallowed and unknown license. Finally a report is generated showing all dependencies and licenses (which can be exported to CSV).

It does not check any latest versions, etc. If you want to have a tool for that I suggest to use [Renovate](https://github.com/renovatebot/renovate)

**Comment by vinodkumar4b9** [comment]
Hi @derkoe ,

Thanks for the detailed information and it is really helpful , Appreciated all the time you have spent to clear my doubts on this.
